scripts/management/slurm_query_status.py

```diff
@@ -23,14 +23,19 @@
 
 
 def get_queued_tasks(user=None):
-    if user != None:
-        cmd = "squeue -A nesi00213 -o '%A %t' -h" + " -u " + user
-    else:
-        cmd = "squeue -A nesi00213 -o '%A %t' -h"
-    process = Popen(shlex.split(cmd), stdout=PIPE, encoding="utf-8")
-    (output, err) = process.communicate()
-    exit_code = process.wait()
-    return output
+    output_list = []
```

Comment:
This needs a to-do to change to treating mahuika and maui jobs separately and linked in to QSW-912